identity-iota/src/tangle/client_map.rs

```diff
@@ -94,6 +101,10 @@ impl ClientMap {
 
     Ok(client)
   }
+
+  pub fn disable_compression(&mut self) {
+    self.compression = false;
+  }
```

Comment:
This should be `set_compression`, otherwise there is no way to re-enable compression in a Client.